tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/UsernamePasswordIntegrationTests.cs

```diff
@@ -61,6 +61,25 @@ public void TestInitialize()
         }
 
         #region Happy Path Tests
+        //[TestMethod]

```

Comment:
you didn't move the unit tests on the CiamAuthorityHelper from Jenny's POC?

Reply:
I migrated them over. They are working as intended.